index.js

```diff
@@ -71,15 +73,18 @@ module.exports = function (options, logger) {
     req[propertyName] = res[propertyName] = id
     res.setHeader(headerName, id)
 
-    req.log.info('request start')
+    if (requestStart) {
+      var reqStartData = { req: req }
+      if (verbose) { reqStartData.res = res }
+      req.log.info(reqStartData, 'request start')
+    }
     res.on('finish', function() {
-      res.log.info(
-          { req: req
-          , res: res
-          , duration: Date.now() - start
-          }
-        , 'request finish'
-        )
+      var reqFinishData = {
+        res: res
+      , duration: Date.now() - start
+      }
```

Comment:
Following my code-style, I would do:

```js
var reqFinishData =
  { res: res
  , duration: Date/now() - start
  }
```

The ide is to put all of `{`, `,` and `}`